pkg/aaparser/aaparser_test.go

```diff
@@ -71,3 +86,33 @@ Copyright 2009-2012 Canonical Ltd.
 		}
 	}
 }
+
+func requiresRoot(t *testing.T) {
+	if !rootEnabled {
+		t.Skip("skipping test that requires root")
+		return
+	}
+	require.Equal(t, 0, os.Getuid(), "This test must be run as root.")
+}
+
+func TestGetVersion(t *testing.T) {
+	output, err := cmd("", "--version")
+	require.NotNil(t, output)
+	version, err := GetVersion()
+	require.NoError(t, err)
+	require.NotEqual(t, -1, version)
+}
+
+func TestLoadProfile(t *testing.T) {
```

Comment:
Agreed, and given that we already have integration test for this in place, I will close this PR.